src/structures/MessageEmbed.js

```diff
@@ -44,7 +44,9 @@ class MessageEmbed {
      * * `gifv` - a gifv embed
      * * `article` - an article embed
      * * `link` - a link embed
+     * <warn>Discord advises that "Embed types should be considered deprecated and might be removed in a future API version."</warn>
```

Comment:
ESLint requires a maximum line length of 120 characters
```suggestion
     * <warn>Discord advises that "Embed types should be considered deprecated
     * and might be removed in a future API version."</warn>
```